erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -249,6 +249,8 @@ def create_stock_entry(self):
 					stock_entry = update_stock_entry_items_for_source(location, stock_entry)
 				if location.package_tag:
 					stock_entry = update_stock_entry_items_for_target(location, stock_entry)
+				else:
```

Comment:
@shantanu-kadam-epi you added else condition for the target package tag when it's not available then it should be handle for the source package tag too.

Reply:
If source package tag is available then the update_stock_entry_items_for_source function get called so need to handle it separately